### PR TITLE
Just refactored some code

### PR DESCRIPTION
**Pull request by ameten**

### CODE REVIEW

File: solc/CommandLineInterface.cpp [hunk 1]
```diff
@@ -65,36 +65,80 @@ namespace dev
...
+static string const g_strOutputDir = "output-dir";
+static string const g_strSignatureHashes = "hashes";
+static string const g_strSources = "sources";
+static string const g_strSourceList = "sourceList";
+static string const g_strSrcMap = "srcmap";
+static string const g_strSrcMapRuntime = "srcmap-runtime";
+static string const g_strVersion = "version";
+static string const g_stdinFileNameStr = "<stdin>";
+
+static string const g_argAbi = g_strAbi;
```
Comment (pirapira) [inline solc/CommandLineInterface.cpp:101]: How is it useful to have two constants of the same value?

Comment (ameten) [reply solc/CommandLineInterface.cpp:101]: just to differentiate between a string constant and option for solc.

Comment (ameten) [reply solc/CommandLineInterface.cpp:101]: not all string constants are solc options.

**Comment by pirapira** [comment]
@federicobond Please rebase.

**Comment by pirapira** [comment]
@ameten please rebase.

**Comment by ameten** [comment]
@pirapira, I have recreated pull request: https://github.com/ethereum/solidity/pull/1491

**Pull request closed without merging**